App.jsx

```diff
@@ -0,0 +1,18 @@
+import React, { useState } from 'react';
+import AreaSubpage from './components/AreaSubpage';
+import Navbar from './components/Navbar';
+import BottomMenu from './components/BottomMenu';
+
+function App(): React.JSX.Element {
+  const [activeColor, setActiveColor] = useState('#FFF');
```

Comment:
The default color should be chosen from the predefined constant colors, otherwise you risk bug where it works first time and later not, or the other way around, using references instead of hard coded values saves time and nerves ;)